src/extensions/default/bramble/lib/ConsoleManager.js

```diff
@@ -15,8 +17,30 @@ define(function (require, exports, module) {
         var args = data.args;
         var type = data.type || "log";
 
-        // TODO: Show this in Custom Console UI, see issue #1675 in Thimble
+        if (type === "error-handler") {
+            var regex = new RegExp('(blob:.+):([^:]+):(.+)', 'gm'),
+                endingRegex = new RegExp(':([0-9]+):([0-9]+)$', 'gm'),
+                stackTrace = args["stack"].split("@"),
+                newArgs = [];
+            
+            // Handle Blob URLs
+            for(var i = 1; i < stackTrace.length; i++){
+                var stackItem = stackTrace[i].match(regex);
+                stackItem = (BlobUtils.getFilename(stackItem[0].replace(endingRegex, ''))).split('/');
```

Comment:
Ah, my mistake in the previous response. I thought you were asking why I was splitting on the `@` symbol (browser window did not show the split at the end of my line you're referring too. 

By not splitting, I'm `stackItem` is assigned the following: 
```js
/7/projects/30/index.html
```

I believe the reason I wanted to split was to separate the information itself. If you prefer I'll remove the `split` function and adjust the output accordingly to display the above. Apologies again for the confusion.